layers/build-krd/dracut/soci/module-setup.sh

```diff
@@ -24,12 +24,16 @@ install() {
     inst_hook initqueue/settled 50 "$moddir/soci-settled.sh"
     inst_script "$moddir/soci-lib.sh" /usr/lib/dracut/soci-lib.sh
 
+    # Signdata is required for secure boot
+    inst "/pcr7data.cpio"
```

Comment:
heh, looks like a "tar" to me =)

Reply:
Originally I'd used tar.  I switched to cpio thinking cpio would be available, while tar wasn't.  Turned out cpio was also not available.  Still cpio is a smaller dependency.